core-bundle/src/EventListener/DataContainer/ResetCustomTemplateListener.php

```diff
@@ -33,9 +33,7 @@ public function __construct(private Connection $connection)
      */
     public function __invoke(mixed $varValue, DataContainer $dc): mixed
     {
-        $type = $dc->getCurrentRecord()['type'] ?? null;
-
-        if ($type === $varValue) {
+        if ($dc->getCurrentRecord()['type'] === $varValue) {
```

Comment:
Because there was no null check before, either:

<img width="505" alt="" src="https://user-images.githubusercontent.com/1192057/176130150-c9b94a0b-4322-41de-a283-7f2bfff9a23e.png">